less/current/layout/searchresultcontainer.less

```diff
@@ -1,5 +1,5 @@
 .search-result-container {
-  background-color: @color-light-gray;
+  background-color: @color-lighter-gray;
```

Comment:
making the background lighter means it's easier to achieve accessible color contrast